guide/english/c/loops/index.md

```diff
@@ -136,7 +136,7 @@ The final section is referred to as the 'increment/decrement'. Its job is to per
 ### Syntax Comparison
 ```
```

Comment:
Could you add the letter C after the triple backticks for syntax  highlighting ?

Thanks